src/platform/radxa/common.c

```diff
@@ -0,0 +1,39 @@
+/*
+	Copyright (c) 2023 Radxa Ltd.
+	Author: Nascs <nascs@radxa.com>
+					ZHANG Yuntian <yt@radxa.com>
```

Comment:
Indentation is off

Reply:
Is it because I'm using a tab size equal to 2 spaces?

Reply:
Hi @CurlyMoo, can you look at this pr again? I've  made the changes.

Reply:
Hi @CurlyMoo, I would appreciate it if you have time to look at this pr again.

Reply:
Hi @CurlyMoo, I hope everything is going well. I really appreciate it that if you could take the time to help review this pr again.